tests/unit/test_protocols.py

```diff
@@ -98,7 +99,12 @@
     'rest-json': RestJSONParser,
     'rest-xml': RestXMLParser,
 }
-PROTOCOL_TEST_BLACKLIST = ['Idempotency token auto fill']
+PROTOCOL_TEST_BLACKLIST = [
+    # These cases test functionality outside the serializers and parsers.
+    "Test cases for QueryIdempotencyTokenAutoFill operation",
+    "Test cases for PutWithContentEncoding operation",
+    "Test cases for HttpChecksumRequired operation",
+]
```

Comment:
We'll want to update usage elsewhere when this is used.

```suggestion
PROTOCOL_TEST_BLOCKLIST = (
    # These cases test functionality outside the serializers and parsers.
    "Test cases for QueryIdempotencyTokenAutoFill operation",
    "Test cases for PutWithContentEncoding operation",
    "Test cases for HttpChecksumRequired operation",
)
```